patches/d9-acms-settings.patch

```diff
@@ -28,7 +28,7 @@ if (AcquiaDrupalEnvironmentDetector::isAhEnv()) {
     ];
     $databases = array_merge_recursive($databases, $default_settings);
     // Only call this function on the cloud, not on a local environment.
-    if (function_exists(acquia_hosting_db_choose_active) {
+    if (function_exists(acquia_hosting_db_choose_active)) {
```

Comment:
@saurabhtripathics I think it should be:
```suggestion
    if (function_exists('acquia_hosting_db_choose_active')) {
```
Ref: https://www.php.net/manual/en/function.function-exists.php

Reply:
done and tested on local @vishalkhode1